source/_docs/scripts.markdown

```diff
@@ -97,6 +97,8 @@ Wait until some things are complete. We support at the moment `wait_template` fo
 
 #### Wait Template
 
+The template is re-evaluated whenever an entity ID that it references changes state. If you use non-deterministic functions like `now()` in the template as it will not be continuously re-evaluated, but only when an entity ID that is referenced is changed. If you need to periodically re-evaluate the template, reference a sensor from the (Time and Date)[/integrations/time_date/] component that will update minutely or daily.
```

Comment:
```suggestion
The template is re-evaluated whenever an entity ID that it references changes state. If you use non-deterministic functions like `now()` in the template it will not be continuously re-evaluated, but only when an entity ID that is referenced is changed. If you need to periodically re-evaluate the template, reference a sensor from the (Time and Date)[/integrations/time_date/] component that will update minutely or daily.
```

Reply:
I am not sure that I like the `time_date` suggestion, it could be strange to fit that into a template. However, I am not sure what to suggest instead.